### PR TITLE
Support nil name & email

### PR DESCRIPTION
**Pull request by ppworks**
If a bot checked 'Keep my email address private', you don't need to know email address. You can assemble email by user name; like a 'user_name@users.noreply.github.com'.

And you can get user_name by access token. So, you'll not need to specify user name & email.

<a href="https://github.com/settings/emails">
private-email</a>

### CODE REVIEW

File: lib/circleci/bundle/update/pr.rb [hunk 1]
```diff
@@ -14,6 +14,10 @@ def self.create_if_needed(git_username: nil, git_email: nil, git_branches: ["mas
           repo_full_name = "#{ENV['CIRCLE_PROJECT_USERNAME']}/#{ENV['CIRCLE_PROJECT_REPONAME']}"
           now = Time.now
           branch = "bundle-update-#{now.strftime('%Y%m%d%H%M%S')}"
+
+          git_username ||= client.user.login
```
Comment (ppworks) [inline lib/circleci/bundle/update/pr.rb]: Fetch user_name via github API.

**Comment by masutaka** [comment]
Thanks a lot!

**Comment by ppworks** [comment]
Thanks for merging "thanks for merging :pray: 
I'm looking forward to v1.2.3(or v.1.3.0?)

**Comment by masutaka** [comment]
@ppworks 
Oops... Just released. Thanks! :ramen: 

https://github.com/masutaka/circleci-bundle-update-pr/releases/tag/v1.3.0
https://rubygems.org/gems/circleci-bundle-update-pr/versions/1.3.0

**Comment by ppworks** [comment]
:tada:

**Pull request merged**